src/collectd.conf.pod

```diff
@@ -2326,6 +2326,12 @@ other interfaces are collected.
 
 =back
 
+=head2 Plugin C<ipc>
```

Comment:
FYI, we usually don't mention plugins which have no configuration options in the manpage.